packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2836,6 +2836,7 @@
   "Message_BadWordsWhitelist": "Remove words from the Blacklist",
   "Message_BadWordsWhitelistDescription": "Add a comma-separated list of words to be removed from filter",
   "Message_Characther_Limit": "Message Character Limit",
+  "Message_Code_highlight": "Message Code highlight",
```

Comment:
I don't think this is explanatory enough to admins..

I'd consider something like "Code languages highlights"
And also add a description to explain better what it does, like "Comma separated list of languages (link to list of supported languages https://github.com/highlightjs/highlight.js/tree/9.18.5#supported-languages ) that will be used to highlight code blocks"